src/Extensions/TwigServiceContainer.php

```diff
@@ -168,9 +169,14 @@ public function getUrl()
     {
         return pluginApp( UrlService::class );
     }
+
+    public function getItemManufacturerService()
```

Comment:
die Zugriffsmethoden in dieser Klasse werden ohne den `Service`-Suffix angeben. Sprich `getItemManufacturer`. Diese Klasse dient lediglich dazu, die Services aus Twig heraus anzusprechen. Dies passiert über die globale Variable `services`. Daher wäre der Suffix an dieser Stelle doppelt.